Modules/CustomRolesHelper.cs

```diff
@@ -145,56 +145,19 @@ static class CustomRolesHelper
-                CustomRoles.Judge => CustomRoles.CrewmateTOHE,
-                CustomRoles.Cleanser => CustomRoles.CrewmateTOHE,
-                CustomRoles.Mortician => CustomRoles.CrewmateTOHE,
-                CustomRoles.Mediumshiper => CustomRoles.CrewmateTOHE,
-                CustomRoles.Bodyguard => CustomRoles.CrewmateTOHE,
-                CustomRoles.ParityCop => CustomRoles.CrewmateTOHE,
-                CustomRoles.Lookout => CustomRoles.CrewmateTOHE,
-                CustomRoles.Grenadier => CustomRoles.EngineerTOHE,
-                CustomRoles.Lighter => CustomRoles.EngineerTOHE,
-                CustomRoles.Transporter => CustomRoles.CrewmateTOHE,
-                CustomRoles.Veteran => CustomRoles.CrewmateTOHE,
-                CustomRoles.GuardianAngelTOHE => CustomRoles.GuardianAngel,
-                CustomRoles.Detective => CustomRoles.CrewmateTOHE,
-                CustomRoles.NiceGuesser => CustomRoles.CrewmateTOHE,
-                CustomRoles.Luckey => CustomRoles.CrewmateTOHE,
-                CustomRoles.CyberStar => CustomRoles.CrewmateTOHE,
-                CustomRoles.TaskManager => CustomRoles.CrewmateTOHE,
-                CustomRoles.Psychic => CustomRoles.CrewmateTOHE,
-                CustomRoles.Needy => CustomRoles.CrewmateTOHE,
-                CustomRoles.SuperStar => CustomRoles.CrewmateTOHE,
-                CustomRoles.Paranoia => CustomRoles.EngineerTOHE,
-                CustomRoles.EngineerTOHE => CustomRoles.EngineerTOHE,
-                CustomRoles.TimeMaster => CustomRoles.EngineerTOHE,
-                CustomRoles.CrewmateTOHE => CustomRoles.CrewmateTOHE,
-                CustomRoles.Doctor => CustomRoles.ScientistTOHE,
-                CustomRoles.ScientistTOHE => CustomRoles.ScientistTOHE,
-                CustomRoles.Tracefinder => CustomRoles.ScientistTOHE,
-                CustomRoles.SpeedBooster => CustomRoles.CrewmateTOHE,
-                CustomRoles.Dictator => CustomRoles.CrewmateTOHE,
-                CustomRoles.Snitch => CustomRoles.CrewmateTOHE,
-                CustomRoles.Marshall => CustomRoles.CrewmateTOHE,
-                CustomRoles.SabotageMaster => CustomRoles.EngineerTOHE,
-                CustomRoles.Retributionist => CustomRoles.Crewmate,
-                CustomRoles.Guardian => CustomRoles.CrewmateTOHE,
-                CustomRoles.Addict => CustomRoles.EngineerTOHE,
-                CustomRoles.Oracle => CustomRoles.CrewmateTOHE,
-                CustomRoles.Chameleon => CustomRoles.EngineerTOHE,
-                _ => role.IsImpostor() ? CustomRoles.ImpostorTOHE : CustomRoles.CrewmateTOHE,
+                RoleTypes.Crewmate => CustomRoles.CrewmateTOHE,
+                RoleTypes.Scientist => CustomRoles.ScientistTOHE,
+                RoleTypes.Engineer => CustomRoles.EngineerTOHE,
+                RoleTypes.Impostor => CustomRoles.ImpostorTOHE,
+                RoleTypes.Shapeshifter => CustomRoles.ShapeshifterTOHE,

```

Comment:
This is where the code defines the role type

Reply:
So what if, for example, Sheriff got erased?
According to the host, their role type is Scientist.
For everyone else, it's Impostor.

Reply:
Hmm, I didn't think this through, I need to make corrections, lol

Reply:
Yeah, that's why I was asking lol.